index.html

```diff
@@ -16,9 +16,10 @@ <h1>Tv Show Project</h1>
       <select name="select" id="selectSeries" class="optional">
```

Comment:
It has a fabulous appearance and works perfect technically. well done Hediyeh. we noticed in another project in week that when we select an episode it shows only one item and then we search something it shows more items. you may remove searchbox when one episode is selected